docs/antora/modules/ROOT/pages/management/docker.adoc

```diff
@@ -53,13 +54,18 @@ Now connect to the Docker container in which you wish to reset the database. If
 $ docker exec -it lisk-service_mysql_core_1 bash # <1>
 ----
 
-<1> Connect to the interactive shell of the MYSQL Docker container in which you wish to reset the MYSQL database.
+<1> Connect to the interactive shell of the MySQL Docker container in which you wish to reset the MySQL database.
 
-=== Drop MYSQL DB
+=== Drop Database
 
+[tabs]
+====
+MySQL DB::
++
+--
 [source,bash]
 ----
-root@f1413b535254:/# mysql -u lisk -ppassword # <1>
+root@f1413b535254:/# mysql -ulisk -ppassword # <1>
```

Comment:
```suggestion
root@f1413b535254:/# mysql -ulisk -ppassword
```

please remove the numbers in the cod snippet, because they are currently not rendered correctly in the docs.

Reply:
OK, I have removed all the numbers from both code snippets 👍